README.md

```diff
@@ -341,6 +341,24 @@ Contribution of more such shared configs for other platforms are welcome!
 
 An array of folders. Resolved modules only from those folders will be considered as "external". By default - `["node_modules"]`. Makes sense if you have configured your path or webpack to handle your internal paths differently and want to considered modules from some folders, for example `bower_components` or `jspm_modules`, as "external".
 
+#### `import/external-modules`
```

Comment:
https://github.com/benmosher/eslint-plugin-import#importexternal-module-folders already exists - how is this different?